packages/react-scripts/config/webpack.config.js

```diff
@@ -193,22 +192,34 @@ module.exports = function (webpackEnv) {
     ].filter(Boolean);
     if (preProcessor) {
       loaders.push(
-        {
-          loader: require.resolve('resolve-url-loader'),
-          options: {
-            sourceMap: isEnvProduction ? shouldUseSourceMap : isEnvDevelopment,
-            root: paths.appSrc,
+        ...[
+          {
+            loader: require.resolve('resolve-url-loader'),
+            options: {
+              sourceMap: isEnvProduction
```

Comment:
Can you undo the automatic formatting here? It makes rebasing upstream difficult in the future.

(My hint here would be to disable any IDE linters when working on BRS just to avoid issues with formatting)

Reply:
Yeah, will restore this.